scripting/upgrades/smrpg_upgrade_conversion.sp

```diff
@@ -0,0 +1,256 @@
+#pragma semicolon 1
+#include <sourcemod>
+
+#pragma newdecls required
+#include <smrpg>
+
+#define UPGRADE_SHORTNAME "conversion"
+#define PLUGIN_VERSION	  "1.0"
+
+public Plugin myinfo =
+{
+	name		= "SM:RPG Upgrade > Conversion ",
+	author		= "WanekWest",
+	description = "Conversion allows the player to turn excess money into exp.",
+	version		= PLUGIN_VERSION,
+	url			= "https://vk.com/wanek_west"
+
+
+}
+
+ConVar g_hCvMoneyConvertType,
+	   g_hCvMoneyConvertRequestAmount, g_hCvMoneyConvertIncreaserPerLevel, g_hCvMoneyConvertBaseAmount;
+
+int	  hCvMoneyConvertType, hCvMoneyConvertRequestAmount;
+float hCvMoneyConvertIncreaserPerLevel, hCvMoneyConvertBaseAmount;
+
+int	  g_MoneyOffset;
+
+public void OnPluginStart()
+{
+	LoadTranslations("smrpg_stock_upgrades.phrases");
+
+	
+	HookEvent("bomb_defused", Event_OnBombDefused);
```

Comment:
Why do you need this huge amount of events? Probably on round end/start you will have full amount of your money and you should work with them.

And by the way all other hooks and e.t.c may cause additional stress during the game. But its only my opinion and how i released that.

Reply:
I wanted it to look like 10000$ is the new maximum amount with the upgrade. The way it translates into the game is that you never see it go above 10000$ with these hooks. Another reason was that it would scale better with bounty addons that would let you get money for killing people. 
If for example someone is worth 5900$, you kill him and another player then you waste 500$ since the game doesn't let you get above 16000.
You're right though that it is not entirely necessary and might be taken out.